skietbaan_fe/src/components/RegisterMemberStyles.css

```diff
@@ -156,16 +164,19 @@
   }
   
   .register-member-user-column{
-    background-color: white;
+    background-color: #798CA5;
+    color: white;
```

Comment:
You are using hex everywhere else please be consistent

Reply:
done